packages/@atjson/document/test/document-test.ts

```diff
@@ -211,20 +211,39 @@ describe("Document#canonical", () => {
       content: "Hello,\n World!",
       annotations: [
         {
-          type: "bold",
+          type: "paragraph",
           start: 0,
-          end: 5,
+          end: 6,
           attributes: {},
         },
         {
-          type: "paragraph",
+          type: "bold",
           start: 0,
-          end: 6,
+          end: 5,
           attributes: {},
         },
       ],
     });
   });
+
+  test("annotations are properly sorted", () => {
```

Comment:
"properly" isn't really descriptive of what properties we expect from this test.

```suggestion
  test("annotations are sorted by position and rank", () => {
```